clouddrift/analysis.py

```diff
@@ -126,6 +127,10 @@ def chunk(
         to offset chunks by some number of elements. For example, if ``length``
         is 2 and ``overlap`` is -1, the chunks of ``[0, 1, 2, 3, 4, 5]`` will
         be ``np.array([[0, 1], [3, 4]])``.
+    discard_last : bool, optional
+        If the number of chunks is not a multiple of the length of ``x``, the
+        points at the end will be thrown away. If ``skip_last`` is False, the
```

Comment:
do you mean if ``discard_last`` is False?

Reply:
Yes, I just fixed it. That was my previous variable name. Also open to any other parameter name by the way!